.github/scripts/check-yaml.py

```diff
@@ -253,12 +247,65 @@ def check(self) -> int:
                             message="The file must be non-empty",
                         )
 
+                if "document" in parsed:
+                    repo = parsed["document"].get("repo")
+                    commit = parsed["document"].get("commit")
+                    patterns = parsed["document"].get("patterns")
+
+                    line = self.get_yaml_line(
```

Comment:
You could end up reporting no `yq` command twice for the same yaml file.

Reply:
right and then we would fix our CI setup ;-)

Reply:
The issue is not on the CI where we already know yq is installed. It is on the user's workstation where they run this script that they may not have yq installed.